source/systems/request.md

```diff
@@ -0,0 +1,43 @@
+
+Thankfully, there’s the Request System! Citizens will automatically ask for the items they need to perform their jobs, and if those items can be provided by your other citizens, they will do so! 
+
+## Getting Started
+
+The request system is based on two components: the [Warehouse](../../source/buildings/warehouse) — which stores all of your colony’s resources — and the [Couriers](../../source/workers/courier) — who pick up and drop off items between buildings. The Couriers wait in the Warehouse until a request comes in.
+
+When a citizen realizes that they need an item, they’ll first look in their inventory, hut block, and attached hut inventory. If they cannot find the item in these locations, they’ll then make a request automatically. If this item is in the Warehouse, a Courier will then take it from the Warehouse and deliver it to your citizen automatically.
+
+## Crafting Requests
+
+But what if you don’t have that item in your Warehouse? In that case, your citizens will check who is able to make that item and attempt to ask them to make it. For example, if your builder needs some oak stairs and there aren’t any in the Warehouse, your [Carpenter](../../source/workers/carpenter) will check if they know how to make oak stairs. If they do, then they’ll request some oak planks to turn into stairs. If there aren’t any oak planks but the carpenter also knows how to make them from oak logs, then they’ll request oak logs to turn into oak planks to turn into oak stairs. Your citizens will keep making and trying to fulfill requests until they have the items they need or they cannot fulfill a request given their jobs and knowledge. As such, it’s convenient to get as many citizens working different jobs as you can to fulfill as many of their requests as they can automatically!
+
+## Manual Requests
+<!-- Put a picture of the red gear for manual requests here!-->
+What if your builder asks for oak stairs but you don’t have a carpenter yet? In that case, the request can’t be filled automatically, and the citizen will wait for you, the mayor, to fulfill the request manually. These citizens will have a red gear over their head and, in talking to them, will tell you what they need. Once you have that item they need, you can give it to them directly or put it in the Warehouse for your Couriers to deliver.
+
+## Clipboard
+
+Of course, being the busy mayor you are, you’ll want to be able to see all of your citizens’ requests without running to each one. For this, there’s the [Clipboard](../../source/items/clipboard). Simply right-click the air with your clipboard and you can see all of your citizens’ requests, no matter where you are!
+
+Some of your citizens will always have constant requests (e.g., the [Smelter](../../source/workers/smelter) requesting ores). Clicking the "Show Important Requests Only" hides these constant requests, so you can focus on the requests that bottleneck your citizens' work.
```

Comment:
maybe directly add the clipboard picture(s) here instead of just linking to its page? same for postbox and stash

Reply:
What would this be better/clearer?